### DISCUSSION TITLE
Mi80 converter help

### DISCUSSION THREAD

**Discussion by Airbrando** [thomluther/ha-anker-solix#66] [General]
Hello everybody. Hopefully someone can help me. Iam from germany and my english is not so good. I have 2 solar plates with a mi80 converter and no battery. My system is directly connect to the house power system. 
The app tells me that my plates are example generating now 500W. How can i show this in home assistant ? When i load the integration from hacs my mi80 is found but i only got 4 entities and nothing is showing my the 500W. Can anybody please help me what i should do ? 
Thank you very much

**Comment by Airbrando** [comment]
Nobody can help ?

**Comment by thomluther** [comment]
Neither the integration nor the Api was ever tested with only Inverter systems. The Api structure for such a system type is unknown, therefore it is not clear how system values can be mapped to device values.
Furthermore if you only get a single device for the inverter but no system device in HA, you have not configured your inverter into a power system for the cloud. Any device values you see in the App are not transmitted to the cloud server, but only directly via Bluetooth or the MQTT server, which is only refreshed when you access device details via the App.

**Comment by thomluther** [comment]
If you have the inverter in a power system that you see on home screen, you need to export your system config at various states to provide examples what you see on home screen and how those values corresponding to the returned Api structure. Then it can maybe be supported in future.
https://github.com/thomluther/anker-solix-api?tab=readme-ov-file#export_systempy
